engine/redis_v2/model/job.proto

```diff
@@ -0,0 +1,8 @@
+syntax = "proto3";
+package model;
+option go_package="github.com/lmstfy/engine/redis_v2/model";
+
+message ExtendedJob {
```

Comment:
We can move this proto file into a dedicated dir and how about renaming it to `JobData` and body => data?

Reply:
Could you please specify "dedicated dir" ? 
 Do you mean under the project root directory?